docs/specs/multi-registry/registries/acme-http-server-lib/metric.yaml

```diff
@@ -0,0 +1,16 @@
+imports:
+  - schema_url: https://opentelemetry.io/schemas/1.27.0
+    name: otel
+
+groups:
+  - ref: otel:http.server.request.duration
```

Comment:
I *think* that reusing the same name across different signals is allowed and is a feature rather than a bug (e.g. `dns.lookup.duration` is a metric name, but if DNS lookup is reported as an event, it'd make a perfect event name, or if it's reported as attribute on something, it could be an attribute name).  

So we need `ref:` to either reference a group id (and then give them some meaning) or reference a specific signal (`ref_metric`).

Reply:
I think this is not something currently allowed or even supported. You’re mentioning in this comment something you’d like us to explore. Is that correct?

Reply:
I'm talking about current state.

I can easily define an attribute with the same name as metric, there are no checks and nothing going to stop me. And I consider it as a feature, not a bug.

Reply:
My bad. It's now fixed. See [5d68d7d](https://github.com/open-telemetry/weaver/pull/348/commits/5d68d7dfb79f88bb90c2318d8e760f69ade5b31c)